### PR TITLE
Import mo form tests

### PR DESCRIPTION
**Pull request by rwlbuis**
Import mo form tests, taken from WebKit and asjusted to wpt style.

### CODE REVIEW

File: mathml/presentation-markup/operators/mo-form-dynamic.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+          var x = document.getElementById("x" + i);
+          x.parentNode.removeChild(x);
+          row.insertBefore(x, row.firstElementChild);
+        }
+      }
+    </script>
+  </head>
+  <body onload="test()">
+
+    <!-- This test verifies that the form of the operators (and thus their spacing) is updated when you change the child list. -->
```
Comment (fred-wang) [inline mathml/presentation-markup/operators/mo-form-dynamic.html]: Maybe this should be in the assert meta tag.

File: mathml/presentation-markup/operators/mo-form-dynamic.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    <link rel="match" href="mo-form-dynamic-ref.html">
+
+    <script>
+      function test()
+      {
+        for (var i = 1; i <= 6; i++) {
+          var row = document.getElementById("row" + i);
+          var x = document.getElementById("x" + i);
+          x.parentNode.removeChild(x);
+          row.insertBefore(x, row.firstElementChild);
```
Comment (fred-wang) [inline mathml/presentation-markup/operators/mo-form-dynamic.html]: Can you make this a reftest-wait test?
(see https://github.com/web-platform-tests/wpt/pull/16575)

File: mathml/presentation-markup/operators/mo-form-dynamic.html [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        }
+      }
+    </script>
+  </head>
+  <body onload="test()">
+
+    <!-- This test verifies that the form of the operators (and thus their spacing) is updated when you change the child list. -->
+    <!-- The test should render the same as the static reference -->
```
Comment (fred-wang) [inline mathml/presentation-markup/operators/mo-form-dynamic.html]: Maybe the pass condition should be text in a normal html paragraph, so people can read it without checking the source.

File: mathml/presentation-markup/operators/mo-form-fallback.html [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+  </head>
+  <body>
+
+    <!-- The left (respectively right) parenthesis only has a prefix
+         (respectively postfix) form which gives its stretchiness.
+         In theory, the parenthesis in the middle of the <math> element are
+         considered infix operators and should not stretch. Let's check that we
+         fallback to the postfix/prefix forms and actually make them stretchy.
+         This is to handle bad MathML markup that misses explicit <mrow> tags.
+      -->
```
Comment (fred-wang) [inline mathml/presentation-markup/operators/mo-form-fallback.html]: I think we need to open an issue in the CG tracker as I don't think that heuristic is standard. Probably the form stuff is only in mathml full not mathml core yet.

**Review by fred-wang — APPROVED** [review]
LGTM. Please remember to update https://github.com/mathml-refresh/mathml-refresh.github.io/wiki/Web-Platform-Tests

File: mathml/presentation-markup/operators/mo-form-dynamic.html (hunk 1 above)
Comment (rwlbuis) [reply mathml/presentation-markup/operators/mo-form-dynamic.html]: Done.

File: mathml/presentation-markup/operators/mo-form-dynamic.html (hunk 2 above)
Comment (rwlbuis) [reply mathml/presentation-markup/operators/mo-form-dynamic.html]: Done.

File: mathml/presentation-markup/operators/mo-form-dynamic.html (hunk 3 above)
Comment (rwlbuis) [reply mathml/presentation-markup/operators/mo-form-dynamic.html]: Done.

**Comment by fred-wang** [comment]
I opened https://github.com/web-platform-tests/wpt/pull/17226 to fix dynamic MathML tests.

I also opened https://github.com/mathml-refresh/mathml/issues/99 for the form heuristics.

@rwlbuis Can you please update https://github.com/mathml-refresh/mathml-refresh.github.io/wiki/Web-Platform-Tests so that we remember to remove the tests from WebKit?

**Pull request merged**